src/client/resources/conversions.ts

```diff
@@ -530,7 +530,8 @@ export function createMigMigration(
   planName: string,
   namespace: string,
   isStage: boolean,
-  disableQuiesce: boolean
+  enableQuiesce: boolean,
+  isRollback: boolean
```

Comment:
this should be optional since this code is shared between stage & migrations

Reply:
We discussed, was a case of editor lagging a bit.